tutorials/matchmaker101/solution/matchfunction/mmf/matchfunction.go

```diff
@@ -17,7 +17,7 @@ package mmf
 import (
 	"fmt"
 	"log"
-	"time"
+	"github.com/google/uuid"
```

Comment:
open match uses this UUID library: github.com/rs/xid. It probably makes sense to be consistent.

Reply:
Agreed. @kongbong Can you make use of that library uuid library and we can close this out

Reply:
I changed it.